### PR TITLE
Updated LinkFree Profile

### PR DESCRIPTION
**Pull request by izzarzn**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1742"></a>

### CODE REVIEW

File: public/data/izzarzn.json [hunk 1]
```diff
@@ -13,6 +13,34 @@
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/izzarzn",
       "icon": "twitter"
+    },
+    {
+      "name": "Check out my website",
+      "url": "izzarzn.github.io",
```
Comment (EmmaDawsonDev) [inline public/data/izzarzn.json:19]:
```suggestion
      "url": "https://izzarzn.github.io",
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Please update the url on line 19

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by izzarzn** [comment]
> Please update the url on line 19
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

Updated

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/izzarzn.json [hunk 2]
```diff
@@ -13,6 +13,34 @@
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/izzarzn",
       "icon": "twitter"
+    },
+    {
+      "name": "Check out my website",
+      "url": "https://izzarzn.github.io",
```
Comment (kumarsonsoff3) [inline public/data/izzarzn.json:19]: There is a broken link, check whether there is a typo, or you've placed the correct URL!!

Please update it with the correct one!! 🛠

image



__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant, Not maintainable

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**